src/components/views/MainPage/MainPage.tsx

```diff
@@ -1,23 +1,33 @@
 import {Text, View} from 'react-native';
 import Logo from '../../../static/imgs/rst_logo.svg';
 import {styles} from './style';
-import {BackArrow, LoginForm, Main, RegisterForm} from '../../../components';
+import {
+  BackArrow,
+  GlobalStateContext,
+  LoginForm,
+  Main,
+  RegisterForm,
+} from '../../../components';
 import {createNativeStackNavigator} from '@react-navigation/native-stack';
 import {useNavigation} from '@react-navigation/native';
 import {globalStyles} from '../../../utils/globalStyles';
-import {useState} from 'react';
+import {useContext, useState} from 'react';
 import {
   ROUTES,
   RootStackParamList,
   WelcomeStackParamList,
 } from '../../../static/types/routeTypes';
 import {StackNavigationProp} from '@react-navigation/stack';
+import {useActor} from '@xstate/react';
 
-const Tab = createNativeStackNavigator<WelcomeStackParamList>();
+const Stack = createNativeStackNavigator<WelcomeStackParamList>();
 
 export const MainPage = () => {
+  const globalServices = useContext(GlobalStateContext);
+  //@ts-ignore
```

Comment:
po co ci to, niepotrzebne i guess

Reply:
jak tego nie mam to krzyczy mi tak
![image](https://github.com/T0ffik/RGPSlashThem/assets/85763715/75a6f3a0-e9d0-4efc-ad9f-a867c4313cd1)
a nie wiem jak tutaj to otypować
![image](https://github.com/T0ffik/RGPSlashThem/assets/85763715/e86902df-7c98-4d54-bb56-c65316b163d7)

Reply:
No jak nie wiesz, jak byk stoi, że generyka tam masz co scr wysłałeś

`const dupa = createContext<Typ>(defaultDupaContext)`